dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -246,7 +248,13 @@ install(){
             export OCM_VERSION=$(ocm list versions --channel-group ${MANAGED_CHANNEL_GROUP} | grep ^${MANAGED_OCP_VERSION})
 	fi
         [ -z ${OCM_VERSION} ] && echo "ERROR: Image not found for version (${version}) on OCM ${MANAGED_CHANNEL_GROUP} channel group" && exit 1
-        ocm create cluster --ccs --provider aws --region ${AWS_REGION} --aws-account-id ${AWS_ACCOUNT_ID} --aws-access-key-id ${AWS_ACCESS_KEY_ID} --aws-secret-access-key ${AWS_SECRET_ACCESS_KEY} --channel-group ${MANAGED_CHANNEL_GROUP} --version ${OCM_VERSION} --multi-az --compute-nodes ${COMPUTE_WORKERS_NUMBER} --compute-machine-type ${COMPUTE_WORKERS_TYPE} --network-type ${NETWORK_TYPE} ${CLUSTER_NAME}
```

Comment:
Just my thought, keep the original ocm create commands and include a new conditional block for custom master types when specified might be a better approach instead of doing this way. We won't be doing custom master deploy much often.

Reply:
+1 I'd like to leave the default there as well and use the other option as a flaggable choice.

Reply:
ACK will add a conditional block.